### ISSUE TITLE
Triplet loss 

### ISSUE THREAD

**Issue by KunpengLi1994** [vlfeat/matconvnet#851]
Hi, I just begin to use MatConvNet. I would like to use Triplet loss in my model. But it still hasn't been included in the MatConvNet. Could you please give me some advises about how to design a Triplet loss layer in MatConvNet with it's implementations in Caffe for reference.Or you guys could share me with a preliminary implementations of the Triplet layer. Thanks! 

Kunpeng

**Comment by Necole123** [comment]
Did you solve this problem? I also want to use Triplet loss in Matconvnet model.

**Comment by lood339** [comment]
I am working on implementing a triplet loss in Matconvnet using the equation 2 in FaceNet paper.
It says loss = (x_a - x_p)^2 - (x_a - x_n)^2 + alpha. What happens when loss < 0.0? Set it to zero, or leave it as it is?
